### PR TITLE
Bug 1969794: Document how to use image registry with a custom PVC backend

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: docs/user/openstack/image_registry_with_custom_pvc_backends.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    ...
+    managementState: Managed
+    pvc:
+      claim: csi-pvc-imageregistry
+...
+```
+
+Second, make sure that the PVC's status is Bound
+
+``sh
```
Comment (mandre) [inline docs/user/openstack/image_registry_with_custom_pvc_backends.md:76]: Missing a backtick
```suggestion
```sh
```

**Review by mandre — COMMENTED** [review]
Looks good, just a small nit.

File: docs/user/openstack/image_registry_with_custom_pvc_backends.md (hunk 1 above)
Comment (Fedosin) [reply docs/user/openstack/image_registry_with_custom_pvc_backends.md]: done!

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**